sda-commons-server-kafka/src/main/java/org/sdase/commons/server/kafka/consumer/strategies/synccommit/SyncCommitMLS.java

```diff
@@ -19,8 +19,18 @@ public class SyncCommitMLS<K, V> extends AutocommitMLS<K, V> {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(SyncCommitMLS.class);
 
+  private final SyncCommitErrorHandler<K, V> syncCommitErrorHandler;
+
   public SyncCommitMLS(MessageHandler<K, V> handler, ErrorHandler<K, V> errorHandler) {
+    this(handler, errorHandler, (Exception e, Consumer<K, V> consumer) -> {});
```

Comment:
To keep the previous behavior until someone adds their own error handler, should we rethrow everything but `CommitFailedException` in the default error handler?